pippo-sasscompiler/pom.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>pippo-parent</artifactId>
+        <groupId>ro.pippo</groupId>
+        <version>0.6.0-SNAPSHOT</version>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+    <packaging>jar</packaging>
+    <artifactId>pippo-sasscompiler</artifactId>
+    <version>0.6.0-SNAPSHOT</version>
+    <name>Pippo Sass</name>
+    <description>Sass css preprocessor</description>
+
+    <dependencies>
+        <dependency>
+            <groupId>ro.pippo</groupId>
+            <artifactId>pippo-core</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.vaadin</groupId>
+            <artifactId>vaadin-sass-compiler</artifactId>
```

Comment:
Are many saas-compilers in java, [lesscss-java](https://github.com/marceloverdijk/lesscss-java) seems to be most popular. Why did you choose `vaadin-sass-compiler`?